### DISCUSSION TITLE
Navabr issue in Mobile view

### DISCUSSION THREAD

**Discussion by v-dale** [primefaces/community#1131] [PrimeVue Blocks]
When I use any Navbar from this list: http://localhost:3100/application/navbar 
when a user clicks on a link, the menu does not hideout. 

This happens if I'm using it outside the router-view:

```vue
<template>
  <NavBar />
  <router-view :key="$route.path" dir="rtl">
  </router-view>
</template>
```

**Comment by tugcekucukoglu** [comment]
Which block do you encounter this issue? https://blocks.primevue.org/application/navbar

**Reply by v-dale** [reply]
I randomly tried 3 of them, all have the same issue. 

I fixed it using this:

```vue
<span>
   <router-link class="p-3" :to="{name:'Home'}"  v-styleclass="{ selector: '.mobileMenu', toggleClass: 'hidden', hideOnOutsideClick: true}">Home</router-link>
</span>
```

But I'm not sure if this is the best way to do it.
